dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ReflectionServiceDescriptor.java

```diff
@@ -74,6 +76,10 @@ private void initMethods() {
         }
 
         methods.forEach((methodName, methodList) -> {
+            //pb method not allow override
+            if(methodList.size() > 1 && methodList.stream().filter(methodDescriptor -> Arrays.stream(methodDescriptor.getParameterClasses()).anyMatch(pclss -> isProtobufClass(pclss))).count() > 0L){
```

Comment:
Please do not add complex judgments to if statements

Reply:
you mean 
`methodList.size() > 1 && methodList.stream()`
need to be split into 2 if judgement

Reply:
Yes.It is better for the stream statement to return a bool value and the if statement to look clearer.

Reply:
ok,i fixed it